code/game/area/areas.dm

```diff
@@ -80,26 +80,26 @@
 	return 0
 
 /area/proc/air_doors_close()
-	if(!src.air_doors_activated)
-		src.air_doors_activated = 1
-		for(var/obj/machinery/door/firedoor/E in src.all_doors)

```

Comment:
Not sure what the point of all_doors was. Perhaps to reduce the size of the loop when closing/opening the air doors? Either way, it was fairly pointless AFAIK so I removed it like /tg/ did a while back.